samples/__tests__/e2e.mediation.test.ts

```diff
@@ -139,7 +129,7 @@ describe('with mediator', () => {
     )
 
     expect(aliceInboundConnection).toBeConnectedWith(mediatorConnectionAtAliceMediator)
-    expect(bobInboundConnection).toBeConnectedWith(mediatorConnectionAtBobMediator)
+    expect(bobInboundConnection).toBeConnectedWith(mediatorConnectionAtBobMediator)*/
```

Comment:
Should we remove this?

Reply:
yes. that will be removed with future commits.